oauth2/oauth2_auth_code_test.go

```diff
@@ -235,7 +235,7 @@ func TestAuthCodeWithDefaultStrategy(t *testing.T) {
 		assert.True(t, i.Get("active").Bool(), "%s", i)
 		assert.EqualValues(t, conf.ClientID, i.Get("client_id").String(), "%s", i)
 		assert.EqualValues(t, expectedSubject, i.Get("sub").String(), "%s", i)
-		assert.EqualValues(t, `{"foo":"bar"}`, i.Get("ext").Raw, "%s", i)
+		assert.EqualValues(t, `bar`, i.Get("ext.foo").String(), "%s", i)
```

Comment:
Changed this to allow for being able to verify ext fields added in the hook tests.